### DISCUSSION TITLE
tutorials/voyage-part2/07-interface-and-entity-definitions

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#64] [course feedback]
> **Note: The course/lesson has been updated; this discussion is outdated.**

# tutorials/voyage-part2/07-interface-and-entity-definitions

https://www.apollographql.com/tutorials/voyage-part2/07-interface-and-entity-definitions

**Comment by kamal** [comment]
I'd like to suggest instead of outright moving fields over, you instead cover how to perform the migration incrementally with no downtime. This means the accounts schema can be published alongside the monolith with the appropriate `@override` annotations.

**Reply by mabuyo** [reply]
Hey there @kamal ! Thanks so much for the suggestion; I looked into the directive and definitely agree that it's the recommended practice. I've added this to our backlog of course updates to tackle.

For anyone curious about what the process of migrating fields and entities using the `@override` directive looks like, you can [refer to the documentation](https://www.apollographql.com/docs/federation/entities-advanced/#migrating-entities-and-fields).
